airtime_mvc/application/controllers/PreferenceController.php

```diff
@@ -262,6 +262,41 @@ public function streamSettingAction()
                     //Application_Model_RabbitMq::PushSchedule();
                 }
 
+                // pulling this from the 2.5.x branch
+                if (!Application_Model_Preference::GetMasterDjConnectionUrlOverride()) {
+                    $master_connection_url = "http://".$_SERVER['SERVER_NAME'].":".$values["master_source_port"].$values["master_source_mount"];
+                    if (empty($values["master_source_port"]) || empty($values["master_source_port"])) {
```

Comment:
I also found [Cool Mic](https://coolmic.net/) which can read a QR code with the URL to make set up easy. Adding such a code is very easy as long as we have a nice URL, Cool Mic would make for a great rewrite of parts of the smartphone journalism docs.